### PR TITLE
Ux updates

### PR DESCRIPTION
**Pull request by catcatherinezhao**
Made UX updates to get the windows in position and shrink them.

### CODE REVIEW

File: client/question/components/LearnerViewDirective.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ tie.directive('learnerView', [function() {
                   <speech-balloons-container></speech-balloons-container>
                 </div>
               </div>
-              <div>
+                <div>
```
Comment (eyurko) [inline client/question/components/LearnerViewDirective.js]: nit - indented?

File: client/question/components/LearnerViewDirective.js [hunk 2]
```diff
@@ -64,6 +64,7 @@ tie.directive('learnerView', [function() {
                 </select>
               </div>
             </div>
+
```
Comment (eyurko) [inline client/question/components/LearnerViewDirective.js]: nit - space not needed

File: client/question/components/LearnerViewDirective.js [hunk 3]
```diff
@@ -117,6 +112,18 @@ tie.directive('learnerView', [function() {
                 </button>
               </div>
             </div>
+
```
Comment (eyurko) [inline client/question/components/LearnerViewDirective.js]: I think this is indented one tab too many also.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,5339 @@
+{
```
Comment (eyurko) [inline package-lock.json]: What is this file?

**Review by eyurko — CHANGES_REQUESTED** [review]
Could be useful to include a screenshot of some kind, as well, but great initial pass! Just a few nits.

File: package-lock.json (hunk 4 above)
Comment (catcatherinezhao) [reply package-lock.json]: I think it was created when I was checking whether node was installed, but I deleted it.

**Review by catcatherinezhao — COMMENTED** [review]
Fixed the spacing/indents, and here's a screenshot of what it looks like currently.

**Review by eyurko — APPROVED** [review]
Nice work!

**Pull request merged**